pkg/builder/webhook.go

```diff
@@ -68,6 +69,12 @@ func (blder *WebhookBuilder) WithValidator(validator admission.CustomValidator)
 	return blder
 }
 
+// WithRecoverPanic takes a bool flag which indicates whether the panic caused by webhook should be recovered.
+func (blder *WebhookBuilder) WithRecoverPanic(recoverPanic bool) *WebhookBuilder {
```

Comment:
How about just defined as `func (blder *WebhookBuilder) RecoverPanic() * WebhookBuilder ` so that `recoverPanic` should be `true` if it is called?